### PR TITLE
Removing Trelis mention

### PR DESCRIPTION
**Pull request by bam241**
converting Trelis mention to Cubit.

This should probably come with a repo renaming

### CODE REVIEW

**Comment by bam241** [comment]
some mention are still present in `scripts` and `mcnp2cad`

the `scripts` one should done after #86 that take care of some of them.

a sperate PR is required for `mcnp2cad`

File: scripts/build_plugin_linux.sh [hunk 1]
```diff
@@ -32,12 +32,12 @@ build_hdf5
 build_moab
 build_dagmc
 
-# $1 is the version of Trelis/Cubit one are trying to compile against i.e. 17.1.0
+# $1 is the version of Cubit one are trying to compile against i.e. 17.1.0
```
Comment (gonuke) [inline scripts/build_plugin_linux.sh:39]:
```suggestion
# $1 is the version of Cubit one is trying to compile against i.e. 17.1.0
```

**Review by gonuke — CHANGES_REQUESTED** [review]
I made one comment outside of this review, but I think this looks OK.  I assume it's failing tests because the repo has not been renamed?

**Comment by bam241** [comment]
> I made one comment outside of this review, but I think this looks OK. I assume it's failing tests because the repo has not been renamed?

Yes:
```
/home/runner/work/Trelis-plugin/Trelis-plugin/scripts/unix_share_build.sh: line 199: cd: Cubit-plugin: No such file or directory
```

**Comment by gonuke** [comment]
Should we announce a plan to rename the repo and then rename it before merging this?

File: README.md [hunk 2]
```diff
@@ -1,19 +1,16 @@
...
 
 **Beta:** This software is currently under early development. It has been
 demonstrated to work on a wide range of problems, but the build system is not
 finalized.
 
-The plugin has been tested and is confirmed to work with Trelis versions 16.5
-and 17.1. These build instructions assume that Trelis 17.1 is being used, but
-other versions can be used by replacing instances of "17.1" with the version
-being used.
+The plugin has been tested and is confirmed to work with various Cubit  versions 17.1 up to 2020.2.
```
Comment (pshriwise) [inline README.md]:
```suggestion
The plugin has been tested and is confirmed to work with various Cubit versions 17.1 up to 2020.2. 
```

File: README.md [hunk 3]
```diff
@@ -32,42 +29,28 @@ built yourself:
...
+Cubit can be installed by obtaining the Cubit `.deb` package and installing it
 with the package manager; i.e.
 
 ```
-sudo dpkg -i Trelis-17.1.0-Lin64.deb
+sudo dpkg -i Cubit_DEB.deb
 ```
 
-This installs Trelis to `/opt/Trelis-17.1`.
+This installs Cubit to `/opt/Coreform-Cubit-VERSION` or `/opt/Trelis-VERSION` for old version.
```
Comment (pshriwise) [inline README.md]: Do we still want the Trelis mention here?

File: README.md [hunk 4]
```diff
@@ -77,13 +60,13 @@ Notes on Build Instructions
 
 A non-source directory build is recommended. These build instructions assume
 that the plugin build will take place in the `${HOME}/plugin-build` directory,
-and they assume that the Trelis-plugin repo has been cloned into
-`${HOME}/plugin-build/Trelis-plugin`.
+and they assume that the Cubit-plugin repo has been cloned into
+`${HOME}/plugin-build/Cubit-plugin`.
 
 **Before building anything, ensure that the `LD_LIBRARY_PATH` environment
 variable is empty**. This variable is not needed to build MOAB, DAGMC, or the
```
Comment (pshriwise) [inline README.md]:
```suggestion
variable is empty**. 

```bash
unset LD_LIBRARY_PATH
```

This variable is not needed to build MOAB, DAGMC, or the
```

File: README.md [hunk 5]
```diff
@@ -92,7 +75,7 @@ MOAB must be built with HDF5 enabled. On Ubuntu 18.04, HDF5 is located in the
 `/usr/lib/x86_64-linux-gnu/hdf5/serial` directory, but it may be located
```
Comment (pshriwise) [inline README.md:78]: This is a ubuntu-specific location for hdf5. We may want to point that out.

File: README.md [hunk 6]
```diff
@@ -207,11 +190,11 @@ cp -pPv ${HOME}/plugin-build/lib/* .
 chmod 644 *
 
 # Set the RPATH to be the current directory for the DAGMC libraries
```
Comment (pshriwise) [inline README.md]: Some explanation of these commands is warranted. Something like:

"The resulting shared library objects require an update to their `RPATH` (runtime path) attribute to ensure that the correct set of libraries is discovered when starting Cubit and loading the plugin library."

**Comment by bam241** [comment]
yes I was thinking on using Thursday meeting for this

**Comment by pshriwise** [comment]
This now has some conflicts after merging #97. I'd be happy to review it as soon as they are resolved.

**Comment by gonuke** [comment]
We also need to stage this for the change in repo name

**Comment by bam241** [comment]
@pshriwise @gonuke this has been rebased.
let's discuss Thursday about the course of actions for this.

File: README.md (hunk 3 above)
Comment (pshriwise) [inline README.md]:
```suggestion
This installs Cubit to `/opt/Coreform-Cubit-VERSION` or `/opt/Trelis-VERSION` for older versions.
```

File: README.md [hunk 7]
```diff
@@ -32,42 +29,28 @@ built yourself:
...
 ```
-sudo dpkg -i Trelis-17.1.0-Lin64.deb
+sudo dpkg -i Cubit_DEB.deb
 ```
 
-This installs Trelis to `/opt/Trelis-17.1`.
+This installs Cubit to `/opt/Coreform-Cubit-VERSION` or `/opt/Trelis-VERSION` for old version.
 
-The Trelis 17 SDK can be installed with these commands:
+For Cubit version older than 2021 one need to manually install the provided SDK
```
Comment (pshriwise) [inline README.md]:
```suggestion
For Cubit versions older than 2021, one needs to manually install the provided SDK
```

**Review by pshriwise — CHANGES_REQUESTED** [review]
Looking pretty good. Just a few suggestions comments from me. Thanks for keeping us up to date, @bam241.

File: README.md (hunk 3 above)
Comment (bam241) [reply README.md]: I kept it because they still mention Trelis 17.1 on the Coreform website, and we still actively support 17.1...

**Comment by bam241** [comment]
@gonuke @pshriwise this is ready for a last run of review or a merge !

**Review by gonuke — APPROVED** [review]

File: README.md (hunk 5 above)
Comment (bam241) [reply README.md:78]: we kinda do the line before, or do you mean something else ?

Comment (pshriwise) [reply README.md:78]: So we do. Must've missed that. My bad!

**Review by pshriwise — APPROVED** [review]
Looks great. Thanks @bam241!

**Pull request merged**